engine/class_modules/sc_death_knight.cpp

```diff
@@ -9039,6 +9129,11 @@ double death_knight_t::composite_player_multiplier( school_e school ) const
     m *= 1.0 + buffs.frenzied_monstrosity->data().effectN( 2 ).percent();
   }
 
+  if ( buffs.final_sentence->up() )
+  {
+    m *= 1.0 + buffs.final_sentence->stack_value();
+  }
+
```

Comment:
Isn't this redundant with the buff's implementation that already makes it affect A_MOD_DAMAGE_PERCENT_DONE?

Unless you meant to put that just below in the pet multiplier, since it seems to be missing from your implementation.